### DISCUSSION TITLE
when do npm run dev first time showing this error

### DISCUSSION THREAD

**Discussion by anishbishnoi127** [vercel/next.js#58807] [Help]
### Summary

 
1 of 1 unhandled error
Next.js is up to date

Unhandled Runtime Error
ChunkLoadError: Loading chunk app/layout failed.
(timeout: http://localhost:3000/_next/static/chunks/app/layout.js)

Call Stack
__webpack_require__.f.j
file:///C:/Users/AnishKumar/Pictures/multi/multi/build/static/chunks/webpack.js (853:29)
<unknown>
file:///C:/Users/AnishKumar/Pictures/multi/multi/build/static/chunks/webpack.js (155:40)
Array.reduce
<anonymous>
__webpack_require__.e
file:///C:/Users/AnishKumar/Pictures/multi/multi/build/static/chunks/webpack.js (154:67)
fn.e
file:///C:/Users/AnishKumar/Pictures/multi/multi/build/static/chunks/webpack.js (391:50)
loadChunk
node_modules\next\dist\compiled\react-server-dom-webpack\cjs\react-server-dom-webpack-client.browser.development.js (202:31)
preloadModule
node_modules\next\dist\compiled\react-server-dom-webpack\cjs\react-server-dom-webpack-client.browser.development.js (128:0)
resolveModule
node_modules\next\dist\compiled\react-server-dom-webpack\cjs\react-server-dom-webpack-client.browser.development.js (1756:0)
processFullRow
node_modules\next\dist\compiled\react-server-dom-webpack\cjs\react-server-dom-webpack-client.browser.development.js (1827:0)
processBinaryChunk
node_modules\next\dist\compiled\react-server-dom-webpack\cjs\react-server-dom-webpack-client.browser.development.js (1975:0)
progress
node_modules\next\dist\compiled\react-server-dom-webpack\cjs\react-server-dom-webpack-client.browser.development.js (2055:0)

### Additional information

```js
1 of 1 unhandled error
Next.js is up to date

Unhandled Runtime Error
ChunkLoadError: Loading chunk app/layout failed.
(timeout: http://localhost:3000/_next/static/chunks/app/layout.js)

Call Stack
__webpack_require__.f.j
file:///C:/Users/AnishKumar/Pictures/multi/multi/build/static/chunks/webpack.js (853:29)
<unknown>
file:///C:/Users/AnishKumar/Pictures/multi/multi/build/static/chunks/webpack.js (155:40)
Array.reduce
<anonymous>
__webpack_require__.e
file:///C:/Users/AnishKumar/Pictures/multi/multi/build/static/chunks/webpack.js (154:67)
fn.e
file:///C:/Users/AnishKumar/Pictures/multi/multi/build/static/chunks/webpack.js (391:50)
loadChunk
node_modules\next\dist\compiled\react-server-dom-webpack\cjs\react-server-dom-webpack-client.browser.development.js (202:31)
preloadModule
node_modules\next\dist\compiled\react-server-dom-webpack\cjs\react-server-dom-webpack-client.browser.development.js (128:0)
resolveModule
node_modules\next\dist\compiled\react-server-dom-webpack\cjs\react-server-dom-webpack-client.browser.development.js (1756:0)
processFullRow
node_modules\next\dist\compiled\react-server-dom-webpack\cjs\react-server-dom-webpack-client.browser.development.js (1827:0)
processBinaryChunk
node_modules\next\dist\compiled\react-server-dom-webpack\cjs\react-server-dom-webpack-client.browser.development.js (1975:0)
progress
node_modules\next\dist\compiled\react-server-dom-webpack\cjs\react-server-dom-webpack-client.browser.development.js (2055:0)
```

### Example

next latest version

**Comment by nicitaacom** [comment]
Sometimes I got this error as well

Just delete .next folder - kill terminal - `npm run dev` again
